e2e-tests/src/tests/studies/ourhealth/home-ui.visual.test.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from '@playwright/test'
+import { test } from 'lib/fixtures/ourhealth-fixture'
+import Home, { Label } from 'pages/ourhealth/home'
+
+test.describe('Home page', () => {
+  test('UI @visual @ourhealth', {
```

Comment:
`@visual` and `@ourhealth` are test tags. Tag is used to select what tests to run with Playwright CLI.